c2corg_ui/templates/sidemenu.html

```diff
@@ -47,39 +41,45 @@
+        <i class="glyphicon glyphicon-comment" tooltip-placement="right" uib-tooltip="{{'Association' | translate}}"></i>
+        <span class="menu-text">Community</span>
       </a>
     </li>
   </ul>
+  
   <footer>
-    <h4 class="text-center" translate>ad</h4>
     <div class="pub-image">
       <img src="data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9InllcyI/PjxzdmcgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB3aWR0aD0iMTQwIiBoZWlnaHQ9IjE0MCIgdmlld0JveD0iMCAwIDE0MCAxNDAiIHByZXNlcnZlQXNwZWN0UmF0aW89Im5vbmUiPjwhLS0KU291cmNlIFVSTDogaG9sZGVyLmpzLzE0MHgxNDAKQ3JlYXRlZCB3aXRoIEhvbGRlci5qcyAyLjYuMC4KTGVhcm4gbW9yZSBhdCBodHRwOi8vaG9sZGVyanMuY29tCihjKSAyMDEyLTIwMTUgSXZhbiBNYWxvcGluc2t5IC0gaHR0cDovL2ltc2t5LmNvCi0tPjxkZWZzPjxzdHlsZSB0eXBlPSJ0ZXh0L2NzcyI+PCFbQ0RBVEFbI2hvbGRlcl8xNTMwOTM2ZThhYyB0ZXh0IHsgZmlsbDojQUFBQUFBO2ZvbnQtd2VpZ2h0OmJvbGQ7Zm9udC1mYW1pbHk6QXJpYWwsIEhlbHZldGljYSwgT3BlbiBTYW5zLCBzYW5zLXNlcmlmLCBtb25vc3BhY2U7Zm9udC1zaXplOjEwcHQgfSBdXT48L3N0eWxlPjwvZGVmcz48ZyBpZD0iaG9sZGVyXzE1MzA5MzZlOGFjIj48cmVjdCB3aWR0aD0iMTQwIiBoZWlnaHQ9IjE0MCIgZmlsbD0iI0VFRUVFRSIvPjxnPjx0ZXh0IHg9IjQ0LjA2MjUiIHk9Ijc0LjU1NjI1Ij4xNDB4MTQwPC90ZXh0PjwvZz48L2c+PC9zdmc+" alt="..." class="img-thumbnail">
     </div>
-    <p translate class="copyright text-center">Site created with the help of:</p>
+    
     <div class="sponsors">
-      <img src="${request.static_url('c2corg_ui:static/img/logo-petzl.png')}">
+      <p translate class="copyright text-center">Site created with the help of:</p>
+      <img src="${request.static_url('c2corg_ui:static/img/logo-petzl.jpg')}">
     </div>
     
     <ul class="footer-nav">
-      <li><span translate>contact</span> </li>
+      <li><span translate>contact</span></li>
       <span class="bullet">&bull; </span>
       <li><span translate>site map</span></li>
       <span class="bullet">&bull; </span>
       <li><span translate>credits</span></li>
+      <span class="bullet">&bull; </span>
       <li><span translate>terms of use</span></li>
       <span class="bullet">&bull; </span>
       <li><span translate>content license</span></li>
     </ul>
-    
+
     <p class="copyright">&copy; 1997-2016 <span translate>all rights reserved</span></p>
     <div class="social-networks">
-      <span class="icon-twitter"></span>
-      <span class="icon-google-plus"></span>
-      <span class="icon-facebook"></span>
-      <a href="http://www.freepik.com/" class="copyright">some icons designed by freepik</a>
+      <div class="icons">
+        <a href="https://twitter.com/camptocamp"><pan class="icon-twitter"></span></a>
+        <a href="https://plus.google.com/104270548458536561874/posts"><span class="icon-google-plus"></span></a>
+        <a href="https://www.facebook.com/Camptocamp-207324185960679/"><span class="icon-facebook"></span></a>
+      </div>
+      <div class="copyright-text">
+        <a href="http://www.freepik.com/" class="copyright">some icons designed by freepik</a>
```

Comment:
Do we want to keep the copyright for the icons in such a prominent place? Can't we only show it in the imprint?

> How to attribute it?
> For websites:
> Insert the attribution on the page of the icon (for example in the page footer) **or on the imprint page**.

http://support.flaticon.com/hc/en-us/articles/207248209-How-I-must-insert-the-attribution-